doc/source/whatsnew/v1.0.0.rst

```diff
@@ -179,6 +179,25 @@ method on a :func:`pandas.api.indexers.BaseIndexer` subclass that will generate
 indices used for each window during the rolling aggregation. For more details and example usage, see
 the :ref:`custom window rolling documentation <stats.custom_rolling_window>`
 
+.. _whatsnew_100.sorting_keys:
+
+Sorting with keys
+^^^^^^^^^^^^^^^^^
+
+We've added a ``key`` argument to the DataFrame and Series sorting methods, including
+:meth:`DataFrame.sort_values`, :meth:`DataFrame.sort_index`, :meth:`Series.sort_values`,
+and :meth:`Series.sort_index`. The ``key`` can be any callable function which is applied
+to the values to determine the sort ordering.
```

Comment:
can you add a reference to the docs.

also add the issue number here.

Reply:
can you do this

Reply:
Sorry this week has been crazy. How do I reference the docs here?